app/controllers/notification/ReceivedALetterController.scala

```diff
@@ -62,11 +62,32 @@ class ReceivedALetterController @Inject()(
         formWithErrors =>
           Future.successful(BadRequest(view(formWithErrors, mode))),
 
-        value =>
+        value => { 
+          
+          val pages = changedPages(request.userAnswers, value)
```

Comment:
We can write this line as:
```val (pagesToClear, hasValueChanged) = changedPages(request.userAnswers, value)```
This means further down we don't have to use pages._1 and pages._2 which aren't very readable

Reply:
resolved!!